### DISCUSSION TITLE
How would I get the "Blog Tags" to show up in the sidebar content?

### DISCUSSION THREAD

**Discussion by nogginfuel** [datenstrom/community#85] [Ask a question] [answered]
I'm trying to tweak the design of the blog and I can seem to wrap my head around swapping out the "Blog Pages" for "Blog Tags". I'd like to only show blog tags so that a user can click on that and see all the articles with that tag.
Do I need to create a file named something like "tags-sidebar.php"? Help is greatly appreciated.

**Comment by markseuffert** [comment, marked as answer]
You can do everything in your content folder, here's an example:

Step 1. Open file `page.txt` in your blog folder, add `Sidebar: sidebar` to the settings at top:

```
---
Title: Blog
Template: blogpages
Description: Blog
Sidebar: sidebar
---
This page is automatically generated.
```

Step 2. Create file `sidebar.txt` in your blog folder, with the following content:

```
---
Title: Sidebar
Status: hidden
---
Tags

[blogtags /blog/]
```

**Comment by nogginfuel** [comment]
That worked like a charm. Thank you again for our fast help. I'm really digging this project.
